worker/src/Channel/UnixStreamSocket.cpp

```diff
@@ -19,18 +19,21 @@ namespace Channel
 	// netstring length for a 4194304 bytes payload.
 	static constexpr size_t NsMessageMaxLen{ 4194313 };
 	static constexpr size_t NsPayloadMaxLen{ 4194304 };
-	static uint8_t WriteBuffer[NsMessageMaxLen];
```

Comment:
Instead of removing this, why not making it thread static?

If not possible (why?) then let's use modern C++ cast instead of `(uint8_t*)std::malloc(NsMessageMaxLen);`. Let's use `<static_cast>` or whatever (I don't know right now). Legacy C style cast produces errors when running `make tidy` in `worker/` folder.

Reply:
First of all: global state is bad, less global state - better.
But more significant reason is that thead_local storage is limited and the same for all threads of the eventual app if I understand it correctly. So by storing there something that could just as well be stored in object's property, we're wasting precious resource.
Sure, it is possible to store it on the heap, but what is the benefit there?